megamek/src/megamek/common/Board.java

```diff
@@ -956,16 +951,10 @@ public void load(final String filename) {
      */
     @Override
     public void load(final File filepath) {
-        // load a board
-        try {
-            java.io.InputStream is = new FileInputStream(filepath);
-            // tell the board to load!
+        try (InputStream is = new FileInputStream(filepath)) {
             load(is);
-            // okay, done!
-            is.close();
-        } catch (java.io.IOException ex) {
-            System.err.println("error opening file to load board!");
-            System.err.println(ex);
+        } catch (IOException ex) {
+            MegaMek.getLogger().error(this, "IO Error opening file to load board! " + ex);
```

Comment:
Curious why you decided to append the ex instead of have it as `MegaMek.getLogger().error(this, "IO Error opening file to load board!", ex);`

Reply:
I just replicated what was done before without much thought. Didn't even notice that ex was the exception, not a message... I'll change it.